test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -859,10 +859,7 @@ private bool ValidateOptions(DeploymentOptions expected, DeploymentOptions actua
                     Assert.True((defaultP == null && actualP == null) 
                      || (defaultP == null && String.IsNullOrEmpty(actualP as string))
                      || defaultP.Equals(actualP)
-                    , $"Actual Property from Service is not equal to default property for {v.Name}, Actual property: {actualP} and Default property: {defaultP}");
-
-                    // Verifying expected and actual deployment options property values are equal
-                    Assert.True((defaultPValue == null && actualPValue == null)
+                     || (defaultPValue == null && actualPValue == null)
```

Comment:
Just making sure I'm reading this correctly: either the properties need to match, OR the property _values_ need to match for the test to pass?

What situation would the properties _not_ match, but the property values would still match?  With your changes, do we still care about the `P` now that we have `PValue`?

Reply:
this assert check is long and hard to read. Can it be simplified like Benjin mentioned?

Reply:
Absolutely! there is no need of bunch of checks that required now as these tests are verifying the default values, I have updated and tested them and converted to simple assertions. Thanks :)